topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.md

```diff
@@ -0,0 +1,767 @@
+
+![Clicking the tool](../../images/wab-tutorial-in-galaxy.png "Tutorial option at the top right in Galaxy")
+
+![Accessing tutorial option within Galaxy](../../images/wab-tutorial-option-filler.png "Click the tool in the tutorial with Galaxy")
+
+> ### {% icon hands_on %} Hands-on: Running Alevin
+>
+> 1. {% tool [Alevin](https://humancellatlas.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/alevin/alevin/1.3.0+galaxy2) %}
+>
+>   > ### {% icon question %} Questions
+>   >
+>   > Try to fill in the parameters of Alevin using what you know!
+>   >
+>   >   > ### {% icon tip %} Tip: Strandedness?
+>   >   >
+>   >   > The Salmon documentation on 'Fragment Library Types' and running the Alevin command (https://salmon.readthedocs.io/en/latest/library_type.html and https://salmon.readthedocs.io/en/latest/alevin.html) will help here, although keep in mind the image there is drawn with the RNA 5' on top, whereas in this scRNA-seq protocol, the polyA is captured by its 3' tail and thus effectively the bottom or reverse strand...)
+>   >   {: .tip}
+>   >
+>   >   > ### {% icon solution %} Solution  
+>   >   >    - *"Select a reference transcriptome from your history or use a built-in index?"*: `Use one from the history`
+>   >   >       - You are going to generate the binary index using your filtered FASTA!
+>   >   >    - {% icon param-file %} *"Transcripts FASTA file"*: `Filtered FASTA`
+>   >   >    - *"Single or paired-end reads?"*: `Paired-end`
+>   >   >    - {% icon param-file %} *"file1"*: `N701-Read1`
+>   >   >    - {% icon param-file %} *"file2"*: `N701-Read2`
+>   >   >    - *"Relative orientation of reads within a pair"*: `Mates are oriented towards each other (IR)`
+>   >   >    - *"Specify the strandedness of the reads"*: `read comes from the reverse strand (SR)`
+>   >   >    - *"Protocol"*: `DropSeq Single Cell protocol`
+>   >   >    - {% icon param-file %} *"Transcript to gene map file"*: `Map`
+>   >   >    - *"Retrieve all output files"*: `Yes`
+>    - In *"Optional commands"*:
+>        - *"dumpFeatures"*: `Yes`
+>        - *"dumpMTX"*: `Yes`
+>   >   {: .solution}
+>   {: .question}
+{: .hands_on}
+
+> ### {% icon comment %} What if I'm running a 10x sample?
+>
+> The main parameter that needs changing for a 10X Chromium sample is the 'Protocol' parameter of Alevin. Just select the correct 10x Chemistry there instead. Additionally, under 'Optional Commands', you can input a 'whitelist file'. This is the file of all the cell barcodes there are in 10x beads. You can get this file here 'https://zenodo.org/record/3457880/files/3M-february-2018.txt.gz' although you ought to check for any changes in chemistry or updates in the whitelist file directly with 10x themselves.
```

Comment:
This is explicitly incorrect- the Alevin docs say the whitelist option is not for supplying barcodes from the 10X lists https://salmon.readthedocs.io/en/latest/alevin.html#whitelist

Reply:
Awesome, even better! Thank you!